features/login-testcase.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Login testcase. 
+
+    Positive scenario (passing valid login and password).
```

Comment:
This has to be posted as a comment (it would crash when you will try to run it). 

So any stuff that is not Gherkin-related, have to be marked as comment. In Python we use "#" for single line comment or """...""" for multiline comment. Please have a look here: https://www.pythonforbeginners.com/comments/comments-in-python

Example:

`# one line comment`

```
"""This
is
multi
line
comment.
"""```